loops.js

```diff
@@ -25,8 +31,18 @@
 // klaar!
 // ==========================================
 
+for (let i = 0; i <5 ; i++) {
+    if (i===4) {
+        console.log("klaar!")
+    } else {
+        console.log("loop...");
+    }
+}
+
+
 
 
+for (let i = 0; i < 5; i++) {    if (i === 4) {        console.log("klaar!")    } else {        console.log("loop...");    }}
```

Comment:
Hier heb ik ook 2 opmerkingen ;-)
1. je hebt na "console.log("klaar!")" geen " ; "gebruikt en na "console.log("loop...");" wel. Dit staat trouwens precies zo in de uitwerking van Novi
2. voor de leesbaarheid is het aan te bevelen om het niet achterelkaar op 1 regel te schrijven.